src/Misc/externals.sh

```diff
@@ -201,15 +213,15 @@ else
         ARCH="darwin-arm64"
         acquireExternalTool "${NODE_URL}/v${NODE16_VERSION}/node-v${NODE16_VERSION}-${ARCH}.tar.gz" node16 fix_nested_dir
         acquireExternalTool "${NODE_URL}/v${NODE20_VERSION}/node-v${NODE20_VERSION}-${ARCH}.tar.gz" node20_1 fix_nested_dir
-    elif [[ "$PACKAGERUNTIME" == "linux-musl-arm64" ]]; then
-        ARCH="linux-arm64-musl"
+    elif [[ "$PACKAGERUNTIME" == "freebsd-x64" ]]; then
+        # Create a link to node only for FreeBSD.
 
-        if [[ "$INCLUDE_NODE10" == "true" ]]; then
-            acquireExternalTool "${CONTAINER_URL}/nodejs/${ARCH}/node-v${NODE10_VERSION}-${ARCH}.tar.gz" node10/bin fix_nested_dir
+        if [[ "$INCLUDE_NODE6" == "true" ]]; then
+            linkExternalTool /usr/local/bin/node node/bin node
```

Comment:
The `/usr/local` part should not be hardcoded but computed from sysctl:

```
LOCALBASE=$(sysctl -n user.localbase)
linkExternalTool "${LOCALBASE}/bin/node" node/bin node
```

Reply:
The `/usr/local` part is now computed from `sysctl`.